apps/src/code-studio/callouts.js

```diff
@@ -71,9 +67,36 @@ module.exports = function createCallouts(callouts) {
     showHideDropletGutterCallouts();
   });
 
-  var showCalloutsMode = document.URL.indexOf('show_callouts=1') !== -1;
 
   $.fn.qtip.zindex = 500;
+  addCallouts(callouts);
+
+  // Insert a hashchange handler to detect triggercallout= hashes and fire
+  // appropriate events to open the callout
```

Comment:
I was curious why we have this code.  An interesting bit of history!  https://github.com/code-dot-org/code-dot-org/pull/8838/files#r66492285